src/api/services/timecardService.js

```diff
@@ -36,3 +36,13 @@ export const getTimePeriodTypes = async (params) => {
     );
   }
 };
+
+export const getTimePeriodTypeById = async (id, params) => {
+  try {
+    return await api.get(baseUrl + 'resources/time-period-type/' + id, params);
```

Comment:
Please can you add this endpoint to the mock server?

(side note: Keeping the mock server in sync with reality is going to be a pain. There must be a better way 🤔)

Reply:
Yes, This should be configurable as env variable (Not the mock server)

Reply:
I've removed this endpoint and the associated tests as it is no longer used